metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -197,6 +207,55 @@ def _exec_col_query(self, table_uri: str) -> Tuple:
 
         return sorted(cols, key=lambda item: item.sort_order), last_neo4j_record
 
+    def _get_type_metadata(self, type_metadata_results: List) -> Optional[TypeMetadata]:
+        """
+        Generates a TypeMetadata object for a column
+
+        :param type_metadata_results: A list of type metadata values for a column
+        :return: a TypeMetadata object
+        """
+        # If there are no Type_Metadata nodes for a column, type_metadata_results
```

Comment:
I think this comment is a bit confusing sine my understanding is that every column (nested or not) has a Type_Metadata node, in this case we would be checking if the column's type metadata nodes has any subtypes right?

Reply:
That's correct that every column would typically have one, but this situation will happen when someone doesn't use the ComplexTypeTransformer and there are no Type_Metadata nodes at all. I see where the confusion comes from so I can clarify the comment